code/modules/projectiles/guns/projectile/automatic/slaught_o_matic.dm

```diff
@@ -0,0 +1,59 @@
+/obj/item/weapon/gun/projectile/automatic/slaught_o_matic
+	name = ".35 slaught-O-matic"
+	desc = "A very cheap self-defence single use automatic pistol. Now less 90% metal. Uses .35 Auto rounds."
+	icon = 'icons/obj/guns/projectile/slaught_o_matic.dmi'
+	icon_state = "slaught"
+	item_state = "slaught"
+	w_class = ITEM_SIZE_SMALL
+	can_dual = 1
+	caliber = CAL_PISTOL
+	max_shells = 0
+	origin_tech = list(TECH_COMBAT = 1, TECH_MATERIAL = 1)
+	slot_flags = SLOT_BELT|SLOT_HOLSTER
+	ammo_type = "/obj/item/ammo_casing/pistol"
+	load_method = MAGAZINE
+	mag_well = MAG_WELL_SMG
+	auto_eject = FALSE
+	magazine_type = /obj/item/ammo_magazine/smg
+	matter = list(MATERIAL_STEEL = 1, MATERIAL_PLASTIC = 9)
+	price_tag = 100
+	rarity_value = 40
+
+	safety = FALSE
+	auto_eject_sound = 'sound/weapons/smg_empty_alarm.ogg'
+	var/list/possible_colors = list("purple", "magenta", "blue", "cyan", "aqua", "green", "yellow", "orange", "red" )
+	var/choosen_color = ""
+
+	damage_multiplier = 0.8
+	penetration_multiplier = 0.2
+	recoil_buildup = 3
+	one_hand_penalty = 5 //despine it being handgun, it's better to hold in two hands while shooting. SMG level.
+
+	init_firemodes = list(
+		list(mode_name="full auto 300",  mode_type = /datum/firemode/automatic, fire_delay=2, icon="burst"),
+		list(mode_name="full auto 800",  mode_type = /datum/firemode/automatic, fire_delay=0.8, icon="auto")
```

Comment:
Please use the defines.

Reply:
If we use FULL_AUTO_300 and FULL_AUTO_800 then the user will not be able to determine which mode is selected.